src/components/Tool/FixCommitmentInfo/FixCommitmentInfo.tsx

```diff
@@ -271,9 +271,7 @@ const FixCommitmentInfo: React.FC<Props> = ({
                       statusValue={contact.status || ''}
                       amount={contact.pledgeAmount || 0}
                       amountCurrency={contact.pledgeCurrency || ''}
-                      frequencyValue={
-                        contact?.pledgeFrequency || PledgeFrequencyEnum.None
-                      }
+                      frequencyValue={contact?.pledgeFrequency || ''}
```

Comment:
When I try to Confirm a contact with no frequency I get this error: `Variable $attributes of type ContactUpdateInput! was provided invalid value for pledgeFrequency (Expected "" to be one of: WEEKLY, EVERY_2_WEEKS, MONTHLY, EVERY_2_MONTHS, QUARTERLY, EVERY_4_MONTHS, EVERY_6_MONTHS, ANNUAL, EVERY_2_YEARS)`

I think we should do `frequencyValue={contact?.pledgeFrequency || null}`